README.md

```diff
@@ -1208,6 +1221,80 @@ GET /v3/apps/:guid?fields=guid,name&fields[droplet]=guid
 }
 ```
 
+
+```json
+GET /v3/service_instances?fields[service_plan.service_offering]=name
```

Comment:
It looks like there are also included resources here. Does using `fields` automatically `include` resources for you? Or would you also need to `include` them manually?

Reply:
We tried to follow the JSON API spec on this one and noticed that there is no definitive specification on what happens when `include` is not passed and `fields[related.resource]`is passed.  Given our use-case of only wanting to include certain fields rather than the entire resource we can choose to implement `fields[related.resource]` without the need for `include`